raindrops/test_raindrops.py

```diff
@@ -0,0 +1,36 @@
+from raindrops import raindrops
+
+
+
+def test_factor_3():
+    assert raindrops(3) == "Pling"
+
+def test_factor_5():
+    assert raindrops(5) == "Plang"
+
+def test_factor_7():
+    assert raindrops(7) == "Plong"
+
+def test_factors_for_number_28():
```

Comment:
`test_number_multiple_of_7`

Reply:
This is not a base case, this and the next 2 are multiples of the base case. Try to communicate the meaning behind what you're testing.